files/openhim/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+ 
+services:
+  mongo:
+    container_name: mongo
+    image: mongo:3.4
+    ports:
+      - "27017:27017"
```

Comment:
If mongo is only going to be used by other docker containers inside the same docker-compose, it's not necessary to expose it on the host :)

(not sure if that's the case).

Reply:
I'll test this without exposing mongo ports and make sure it works. If so, I'll update this PR.